test/cppsuite/src/main/collection.cpp

```diff
@@ -26,20 +26,24 @@
  * OTHER DEALINGS IN THE SOFTWARE.
  */
 
-#include "scoped_connection.h"
-
-#include "connection_manager.h"
+#include "collection.h"
 
 namespace test_harness {
-
-scoped_connection::scoped_connection(const std::string &db_conn_config, const std::string &home)
+/* collection class implementation */
```

Comment:
Remove this comment. I think it's was used to separate class definitions when we had many classes in the same file.

Reply:
See 0fe91dfeda663411b755bf92f6086ebedb688220